invenio_pidstore/cli.py

```diff
@@ -153,3 +164,33 @@ def dereference_object(object_type, object_uuid, status):
         click.echo(
             '{0.pid_type} {0.pid_value} {0.pid_provider}'.format(found_pid)
         )
+
+
+@pid.command()
+@click.argument('minter')
+@click.argument('ids', nargs=-1, required=True)
+@click.option('-l', '--loader',
+              default='invenio_records.api:Record.get_record',
+              callback=loader_import)
+@with_appcontext
+def mint(minter, ids, loader):
+    """Mint objects."""
+    try:
```

Comment:
like this? https://github.com/inveniosoftware/invenio-pidstore/pull/80/files#diff-bc7ea72a8f254847e346dcce4275ddbfR51